### ISSUE TITLE
ESLint couldn't find the config "plugin:ember/gts-recommended" to extend from

### ISSUE THREAD

**Issue by c0rydoras** [ember-cli/eslint-plugin-ember#1989]
eslint is unable to find `plugin:ember/gts-recommended` and `plugin:ember/gjs-recommended`

| package | version |
| --- | --- |
| eslint | ^8.52.2 |
| eslint-plugin-ember | `master` |

```
yarn run v1.22.19
$ eslint .

Oops! Something went wrong! :(

ESLint: 8.52.0

ESLint couldn't find the config "plugin:ember/gts-recommended" to extend from. Please check that the name of the config is correct.

The config "plugin:ember/gts-recommended" was referenced from the config file in "/<project-directory>/.eslintrc.js".

If you still have problems, please stop by https://eslint.org/chat/help to chat with the team.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

eslintrc.js (from readme)
```js
// .eslintrc.js
module.exports = {
  overrides: [
    {
      files: ['**/*.{js,ts,gjs,gts}'],
      plugins: ['ember'],
      extends: [
        'eslint:recommended',
        'plugin:ember/recommended', // or other configuration
      ],
      rules: {
        // override / enable optional rules
        'ember/no-replace-test-comments': 'error'
      }
    },
    {
      files: ['**/*.gts'],
      parser: 'eslint-plugin-ember/gjs-gts-parser',
      plugins: ['ember'],
      extends: [
        'eslint:recommended',
        'plugin:ember/gts-recommended', // or other configuration
      ],
    },
    {
      files: ['**/*.gjs'],
      parser: 'eslint-plugin-ember/gjs-gts-parser',
      plugins: ['ember'],
      extends: [
        'eslint:recommended',
        'plugin:ember/gjs-recommended', // or other configuration
      ],
    },
    {
      files: ['tests/**/*.{js,ts,gjs,gts}'],
      rules: {
        // override / enable optional rules
        'ember/no-replace-test-comments': 'error'
      }
    },
  ],
};
```

**Comment by patricklx** [comment]
Ah, it actually should be `recommended-gts`

**Comment by c0rydoras** [comment]
I opened a PR: https://github.com/ember-cli/eslint-plugin-ember/pull/1990

**Comment by NullVoxPopuli** [comment]
fix merged! thanks for reporting and fixing!
